homeassistant/components/zeroconf/__init__.py

```diff
@@ -87,7 +87,7 @@
 )
 
 
-class HaServiceInfo(TypedDict):
+class ZeroconfServiceInfo(TypedDict):
```

Comment:
Or maybe we could use `ZeroconfDiscoveryInfo` to match the generic `DiscoveryInfoType`?
```suggestion
class ZeroconfDiscoveryInfo(TypedDict):
```

Reply:
I think the current suggestion (`ZeroconfServiceInfo`) is fine. It matches what is represented in Zeroconfig itself.

Reply:
Should I wait for @bdraco to confirm that he too prefers just `ServiceInfo`?
I've marked the other PRs as draft until this one gets merged.

Reply:
I like the ZeroconfServiceInfo name because there are some integrations that have multiple discovery sources and it will easier to differentiate with something like SSDPServiceInfo, DHCPserviceInfo.

Reply:
I'm ok with that :)
Do you approve otherwise?